### PR TITLE
feat(errors): support localized throttled message

### PR DESCRIPTION
**Pull request by vladikoff**
Related to bz1093856

Fixes https://github.com/mozilla/fxa-bugzilla-mirror/issues/48

Ref https://github.com/mozilla/fxa-bugzilla-mirror/issues/48

### CODE REVIEW

**Comment by vladikoff** [comment]
@vbudhram @shane-tomlinson r?

**Comment by vladikoff** [comment]
### For maximum fun this requires: https://github.com/mozilla/fxa-auth-server/pull/1453

image

**Comment by vladikoff** [comment]
With translations: 
image

**Comment by vladikoff** [comment]
node 0.10 tests failed because the auth server doesn't work with node 0.10 anymore :o

**Comment by shane-tomlinson** [comment]
> node 0.10 tests failed because the auth server doesn't work with node 0.10 anymore :o

🎆

File: app/scripts/lib/auth-errors.js [hunk 1]
```diff
@@ -341,6 +346,15 @@ define(function (require, exports, module) {
           return {
             property: err.property
           };
+        } else if (this.is(err, 'THROTTLED')) {
+          if (err.retryAfterLocalized) {
+            // enhance the throttled message if localized retryAfter data is available
+            err.message = THROTTLED_LOCALIZED_ERROR_MESSAGE;
```
Comment (shane-tomlinson) [inline app/scripts/lib/auth-errors.js]: Ah, that's a nice solution.

**Comment by shane-tomlinson** [comment]
I'm waiting for the functional tests to run locally. r+ barring any failures.

**Pull request merged**